test/reactTags.test.js

```diff
@@ -245,6 +245,29 @@ describe('Test ReactTags', () => {
     $input.simulate('keyDown', { keyCode: DOWN_ARROW_KEY_CODE });
   });
 
+  test('disables the input with disabled prop', () => {
+    const $el = mount(
+      mockItem({
+        disabled: true,
+      })
+    );
+
+    const $input = $el.find('.ReactTags__tagInputField');
+
+    expect($input.prop('disabled')).to.be.true;
+  });
+
+  test('default placeholder text is different when disabled', () => {
+    const $el = mount(
+      mockItem({
+        disabled: true,
+      })
+    );
+
+    const $input = $el.find('.ReactTags__tagInputField');
+    expect($input.prop('placeholder')).to.equal('This input field is disabled');
+  });
+
```

Comment:
Can you also add a test inside the describe block for `disabled`
* when disabled prop is false, default placeholder is shown
* when placeholder prop is passed, placeholder prop is shown